modules/crio_arch.adoc

```diff
@@ -0,0 +1,56 @@
+= Understanding CRI-O
+
+The link:http://cri-o.io[CRI-O] container engine provides a stable, more secure, and performant
+platform for running OCI (link:https://www.opencontainers.org/[Open Container Initiative])
+compatible runtimes.
+The CRI-O container engine can be used to launch containers and pods
+via OCI-compliant runtimes like runc, the default OCI runtime, or
+link:https://katacontainers.io/[Kata Containers].
+CRI-O's purpose is to be the container engine implementing
+the Kubernetes Container Runtime Interface (CRI) for {product-title} and Kubernetes,
+replacing the Docker service.
+
```

Comment:
Maybe move the _CRI-O is not supported as a stand-alone container engine_ paragraph from _modules/crio_get.adoc_ to here? See https://github.com/openshift/openshift-docs/pull/12794/files#r231215637

Reply:
I'm going to leave it. When someone goes to get CRI-O I want to make the point that they can only get it (in a supported way) by installing OpenShift or adding a CRI-O node to an existing OpenShift cluster.